tests-server/index.html

```diff
@@ -25,22 +25,27 @@
     const list = document.getElementById('list')
 
     function onUpdate(books) {
-      const l = []
-      books['hydra:member'].forEach((book) => {
-        const li = document.createElement('li')
-        li.dataset.testid = 'book'
-        li.innerText = `${book.title} - ${book.author?.name}`
-        l.push(li)
+      books['hydra:member'].forEach((book, i) => {
+        // Emulate loading time with a random timeout
+        const delay = (i + 1) * Math.floor(Math.random() * (750 - 250) + 250);
```

Comment:
I actually use 3G slow to show this but indeed it's pretty

Reply:
I meant that it's impossible to do this in the code, it has to be done manually in the browser. If we could automate this later on in tests, this could be fine.